committeeoversightapp/views.py

```diff
@@ -249,7 +278,7 @@ def post(self, request, **kwargs):
             save_documents(event, transcript_form.cleaned_data)
             save_witnesses(event, witness_formset.cleaned_data)
 
-        return redirect('list-event')
+        return redirect('/hearings')
```

Comment:
Similarly, should this be a hardcoded path or a reference to a view name?

Reply:
The `/hearings` page is now created in Wagtail, since we need the intro text to be editable, so it doesn't have a view slug.

Reply:
Oh interesting! Seems like an important limitation of Wagtail that I wasn't aware of.